policyengine_us/parameters/gov/hhs/chip/fcep/income_limit.yaml

```
@@ -0,0 +1,111 @@
+description: State governments limit FCEP CHIP pregnancy eligbiilty to these incomes. 
```

Avoid abbreviations in descriptions
```suggestion
description: State governments limit Children's Health Insurance Program from-conception-to-end-of-pregnancy eligibility to these incomes, as a share of the federal poverty line. 
```